bundle/terranova/tnresources/database_catalog.go

```diff
@@ -0,0 +1,65 @@
+package tnresources
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databricks/cli/bundle/config/resources"
+	"github.com/databricks/cli/bundle/deployplan"
+	"github.com/databricks/cli/libs/structdiff"
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/service/database"
+)
+
+type ResourceDatabaseCatalog struct {
+	client *databricks.WorkspaceClient
+	config database.DatabaseCatalog
+}
+
+func (d ResourceDatabaseCatalog) Config() any {
+	return d.config
+}
+
+func (d ResourceDatabaseCatalog) DoCreate(ctx context.Context) (string, error) {
+	fmt.Printf("Creating catalog with name=%s", d.config.Name)
+	response, err := d.client.Database.CreateDatabaseCatalog(ctx, database.CreateDatabaseCatalogRequest{
+		Catalog: d.config,
+	})
+	if err != nil {
+		return "", SDKError{Method: "Database.CreateDatabaseCatalog", Err: err}
+	}
+	return response.Name, nil
+}
+
+func (d ResourceDatabaseCatalog) DoUpdate(ctx context.Context, oldID string) (string, error) {
+	panic("updating a database catalog is not yet supported")
```

Comment:
Not supported by DABs or by API?

If latter, please return an error. If former and you plan to add support later, the error is still recommended but not required.